ppml/trusted-big-data-ml/scala/docker-occlum/start-occlum-attestation-server.sh

```diff
@@ -0,0 +1,18 @@
+# Clean up old container
+sudo docker rm -f bigdl-ppml-trusted-big-data-ml-scala-occlum-attestation-server
+
+# Run new command in container
+sudo docker run -itd \
+	--net=host \
+	--name=bigdl-ppml-trusted-big-data-ml-scala-occlum-attestation-server \
+	--cpuset-cpus 5-7 \
+	--device=/dev/sgx/enclave \
+	--device=/dev/sgx/provision \
+	-v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+	-e LOCAL_IP=$LOCAL_IP \
+	-e SGX_MEM_SIZE=8GB \
+	-e PCCS_URL=$PCCS_URL \
+	-e ATTESTATION_SERVER_IP=$ATTESTATION_SERVER_IP \
+	-e ATTESTATION_SERVER_PORT=$ATTESTATION_SERVER_PORT \
+	intelanalytics/bigdl-ppml-trusted-big-data-ml-scala-occlum:2.1.0-SNAPSHOT \
+	bash
```

Comment:
bash - > run_attestation_server.sh ?

Reply:
updated in this https://github.com/intel-analytics/BigDL/pull/4507/files#diff-d7c7a18415f078d25eceb39b1f3e80a4f99694eafdc07cdd4401b2433afe9a24R18